code/modules/mob/living/simple_animal/hostile/alien.dm

```diff
@@ -145,3 +145,25 @@
 		dead = 1
 		visible_message("[src] lets out a waning guttural screech, green blood bubbling from its maw...")
 		playsound(src, 'sound/voice/hiss6.ogg', 100, 1)
+
+
+/mob/living/simple_animal/hostile/alien/maid
+	name = "lusty xenomorph maid"
+	melee_damage_lower = 0
+	melee_damage_upper = 0
+	a_intent = "help"
+	friendly = "caresses"
+	environment_smash = 0
+	icon_state = "maid"
+	icon_living = "maid"
+	icon_dead = "maid_dead"
+
+/mob/living/simple_animal/hostile/alien/maid/AttackingTarget()
+	if(istype(target, /atom/movable))
+		if(istype(target, /obj/effect/decal/cleanable))
+			visible_message("[src] cleans up \the [target].")
+			qdel(target)
+			return
+		var/atom/movable/M = target
+		M.clean_blood()
+		visible_message("[src] polishes \the [target].")

```

Comment:
`"lusty xenomorph maid polishes Baldie McGriefer."`
improper capitalization, use either `\the [src]` or change the name to `Lusty Xenomorph maid`
also, lewd.